README.md

```diff
@@ -25,3 +25,15 @@ To run the tests:
 ```bash
 npm test
 ```
+
+## Test app
+
+This is very much a work in progress.
+
+To run the test app:
+
+```
+$ cd test/app
```

Comment:
I'm not massively bothered where it goes, i just thought it would be a nice place as it is a test thing, it's just not automated tests. What do you suggest?